README.md

```diff
@@ -1,32 +1,47 @@
-# RebelAllianceBank
+# Rebel Alliance Bank  
 
-A short and clear description of what the project is and what it does.
+## Description  
+Rebel Alliance Bank is an internet bank designed to provide customers with a seamless and user-friendly platform to manage their day-to-day finances.  
 
-## Innehållsförteckning
+The application enables:  
+- Effortless money transfers, either between your own accounts or to external accounts.  
+- Taking out loans or repaying existing loans.  
 
-1. [Description](#descripton)
-2. [Installation](#installation)
-3. [Användning](#användning)
-4. [Funktioner](#funktioner)
-5. [Bidra](#bidra)
-6. [Licens](#licens)
-7. [Kontakt](#kontakt)
+Built with C#, the application offers a structured and scalable foundation, making it easier to implement future enhancements. While the application is functional, some bugs may still be present; we are committed to addressing these in upcoming patches.  
+
+### Planned Features  
+We aim to implement the following features in future updates:  
+- A transaction queue system to introduce a slight delay (e.g., a few minutes) for processing transfers.  
```

Comment:
already implemented :)

Reply:
yea i did this pr 3 days ago so understandeble :)